bindings/python/pyiree/compiler2/tools.py

```diff
@@ -149,7 +149,9 @@ def invoke_immediate(command_line: List[str],
       run_args["input"] = immediate_input
 
     # Capture output.
-    run_args["capture_output"] = True
+    # Upgrade note: Python >= 3.7 can just use capture_output=True
```

Comment:
There area few other instances of things like this in the codebase (including another for exactly this issue). I'll throw together a github issue to group them together.

Reply:
+1